doc/services/tracing/index.rst

```diff
@@ -382,8 +382,8 @@ all initialized mutexes, one can write::
 
```

Comment:
Ok, but there are no trackers (or tracing) for zync :(

Isn't this downgrading Zephyr debug capabilities?

Reply:
I don't have strong feelings, but I kinda think it's important to have tracing/tracking available from the get-go, as this is new stuff that may cause mysterious and subtle bugs on certain workloads, and having a nice way to debug zync behaviour would be really handy. Don't you think?
But I maybe overestimating tracing/tracking importance here...

Reply:
This needs to be reenabled as part of this PR or whatever removes it. We can't be dropping features with the promise to add them later.